core/app/models/spree/store_credit.rb

```diff
@@ -123,25 +128,35 @@ def void(authorization_code, options = {})
     end
   end
 
-  def credit(amount, authorization_code, order_currency, options = {})
-    # Find the amount related to this authorization_code in order to add the store credit back
-    capture_event = store_credit_events.find_by(action: CAPTURE_ACTION, authorization_code: authorization_code)
+  def debit(amount, options = {})
+    # make sure debit amounts are stored as negative number
+    amount = -1 * amount unless amount < 0
```

Comment:
I found this just slightly tough to parse (have to flip the sign because of the unless). I'd prefer

``` ruby
amount = -amount if amount > 0
```